internal/gui/gui.go

```diff
@@ -417,6 +438,24 @@ func handleUserInteraction(conf *visorconfig.V1, doneCh chan<- bool) {
 	}
 }
 
+func handleRootInteraction(conf *visorconfig.V1, doneCh chan<- bool) {
+	for {
+		select {
+		//		case <-mOpenHypervisor.ClickedCh:
```

Comment:
Dead code?

Reply:
changed in [4de574f](https://github.com/skycoin/skywire/pull/1155/commits/4de574fc6930e038a1b9759635667b512b1596f1)